### ISSUE TITLE
Block certain apps for getting updated in "winget upgrade --all"

### ISSUE THREAD

**Issue by TerraDev64** [microsoft/winget-cli#1135]
# Description of the new feature/enhancement

A block list for apps that the user doesn't want to get updated when using "winget upgrade --all"

# Proposed technical implementation details (optional)

A json or xml list where the user can add apps that shouldn't get updated.
A command where the user can add the app name for blocking it form updating/unblocking.

**Comment by RichiCoder1** [comment]
I believe the spirit of this is technically a duplicate of #476

**Comment by denelon** [comment]
Duplicate of #476
